var/spack/repos/builtin/packages/wcs/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Wcs(CMakePackage):
+    """Simulates whole cell models using discrete event simulation."""
+
+    homepage = "https://github.com/LLNL/wcs.git"
+    git      = "https://github.com/LLNL/wcs.git"
+    maintainers = ['rblake-llnl']
+
+    version('master', branch='master')
+    version('devel', branch='devel')
```

Comment:
```suggestion
    version('develop', branch='devel')
```
This will allow the version to sort correctly